content/the-ember-times-issue-175.md

```diff
@@ -11,22 +12,26 @@ tags:
 ---
 
 👋 Emberistas! 🐹
-
+Read the RFC on official TypeScript support on Ember ✅,
 `ember-responsive-image` v2 released 🖼️,
 Use Mirage JS to test non-trivial user flows ⬆️,
 ...
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 
 ---
 
-## [1. Section title in sentence case 🐹](section-url)
+## [RFC: Official TypeScript support ✅](https://github.com/emberjs/rfcs/pull/724)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Chris Krycho (@chriskrycho)](https://github.com/chriskrycho) put up an [RFC for Official TypeScript support](https://github.com/emberjs/rfcs/pull/724) in Ember. The goal of the RFC is to determine if we should support [TypeScript](https://www.typescriptlang.org/) as a first-class citizen in Ember, and if yes, identify any gaps present today.
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+> Ember officially supporting TypeScript means: All libraries which are installed as part of the default blueprint must ship accurate and up-to-date type definitions for the current edition. These types will uphold a Semantic Versioning commitment which includes a definition of SemVer for TypeScript types as well as a specification of supported compiler versions and settings, so that TypeScript will receive the same stability commitments as the rest of Ember.
+
+- Any addon that is part of the default blueprint must ship types.
+- The RFC explicitly does not require support for “typed templates”
```

Comment:
I'd love it if we clarified here that it's also a thing the community is actively working on. It's good to be clear that we aren't *blocked* on it, but we also think it's absolutely key to the long-term experience of TS in Ember!